base/iterators.jl

```diff
@@ -945,6 +945,34 @@ IteratorSize(::Type{<:DropWhile}) = SizeUnknown()
 eltype(::Type{DropWhile{I,P}}) where {I,P} = eltype(I)
 IteratorEltype(::Type{DropWhile{I,P}}) where {I,P} = IteratorEltype(I)
 
+"""
+    findeach(f, it)
+    findeach(it)
+
+An iterator that generates every key from the key/value pairs of `pairs(it)`,
+where `f(value)` returns `true`.
+
+If `f` is not specified, default to `identity`.
+
+`Iterators.findeach` is the lazy equivalent of `findall`.
+
+!!! compat "Julia 1.13"
+    Lazy `findeach` requires at least Julia 1.13.
```

Comment:
The "lazy" qualifier here makes it sound like there's a differently behaving `findeach` in another version.
```suggestion
    `findeach` requires at least Julia 1.13.
```